content/en/tracing/trace_collection/otel_instrumentation/java.md

```diff
@@ -19,22 +19,62 @@ further_reading:
 
 The following OpenTelemetry features implemented in the Datadog library as noted:
 
-| Feature                               | Support notes                       |
-|---------------------------------------|-------------------------------------------|
-| [OpenTelemetry Context propagation][1]         | [Datadog distributed header format][9] is used instead. | 
-| [Span processors][2]                  | Unsupported                                          | 
-| [Span Exporters][3]                   | Unsupported                                            |
-| Trace/span [ID generators][4]         | ID generation is performed by `ddtrace`.           |
-
+| Feature                                           | Support notes                                                      |
+|---------------------------------------------------|--------------------------------------------------------------------|
+| [OpenTelemetry Context propagation][1]            | [Datadog distributed header format][9] is used instead by default. | 
+| [Span processors][2]                              | Unsupported                                                        | 
+| [Span Exporters][3]                               | Unsupported                                                        |
+| Trace/span [ID generators][4]                     | ID generation is performed by `ddtrace`.                           |
+| [Metrics][7], [Baggage][12] and [Context][10] API | Unsupported                                                        |
+| Instrumentation [annotations][6]                  | Unsupported                                                        |
 
 ## Configuring OpenTelemetry to use the Datadog tracing library
 
 1. Add your desired manual OpenTelemetry instrumentation to your Java code following the [OpenTelemetry Java Manual Instrumentation documentation][5].
+
 1. Add the [Datadog tracing library to the JVM][11].
+
+1. Make sure you only depends from OpenTelemetry API (and not OpenTelemetry SDK).
```

Comment:
```suggestion
1. Make sure you only depend on the OpenTelemetry API (and not OpenTelemetry SDK).
```
I think this is what you meant to say? ^

Reply:
Yes, I think so. Thanks!